tests-node/unit/broccoli/translation-reducer/index-test.js

```diff
@@ -203,6 +203,23 @@ describe('translation-reducer', function() {
       ]);
     });
 
+    it('lintTranslations returns list of icu argument mismatch without those that are required', function () {
+      let subject = new TranslationReducer(this.input.path(), { fallbackLocale: 'en'});
+
+      subject.options.requiresTranslation = (_, locale) => locale === 'en';
```

Comment:
The implementation I was thinking of was avoid reporting on ICU Argument Mismatching if the translation came from the fallback locale.  I don't believe `requiresTranslation` should play a part in this implementation.

Let me know if you have thoughts on this.

Reply:
I agree with that and it's gonna be even simpler that way. Please take a look at recent changes then 👍